Mage/src/main/java/mage/cards/CardImpl.java

```diff
@@ -807,25 +807,55 @@ public boolean addCounters(Counter counter, UUID playerAddingCounters, Ability s
     }
 
     @Override
-    public void removeCounters(String name, int amount, Ability source, Game game) {
+    public void removeCounters(String name, int amount, Ability source, Game game){
+        removeCounters(name, amount, source, game, false);
+    }
+
+    public void removeCounters(String name, int amount, Ability source, Game game, boolean damage) {
```

Comment:
1. New method must be defined in Card class;
2. Old method must be implemented as default and final in Card class (so inherited classes must implement only new method). Add new param to all old implementations;
3. use `isDamage` name, not damage;